rdflib/graph.py

```diff
@@ -394,11 +388,14 @@ def add(self, triple):
     def addN(self, quads):
         """Add a sequence of triple with context"""
 
-        self.__store.addN((s, p, o, c) for s, p, o, c in quads
-                          if isinstance(c, Graph)
-                          and c.identifier is self.identifier
-                          and _assertnode(s,p,o)
-                          )
+        triples_stored = self.__store.addN((s, p, o, c) for s, p, o, c in quads
+                                           if isinstance(c, Graph)
+                                           and c.identifier is self.identifier
+                                           and _assertnode(s, p, o)
+                                           )
+        if triples_stored is None:
+            return -1
+        return triples_stored
```

Comment:
hmm, this won't really work without also at least updating the store interface in `store.py`

also i'd prefer a simple `return` in front of the dispatch to store... i don't see how `-1` is better than simply returning `None`

Reply:
You're correct. I don't actually recall why I went with -1.
The intention is to make it optional. I have added a flag member variable to indicate whether addN returns a count on a store I maintain, but I didn't see a reason to make it official interface.